packages/theme-predictive-search/src/theme-predictive-search.js

```diff
@@ -10,18 +10,31 @@ export default function PredictiveSearch(config) {
     throw new TypeError("No config object was specified");
   }
 
+  var configParameters = config;
+
   this._retryAfter = null;
   this._currentQuery = null;
 
   this.dispatcher = new Dispatcher();
   this.cache = new Cache({ bucketSize: 40 });
-  this.configParams = objectToQueryParams(config);
+
+  this.searchUrl = "/search/suggest.json";
+
+  if(configParameters.search_url) {
+    this.searchUrl = configParameters.search_url;
+    delete configParameters['search_url'];
+  }
```

Comment:
This is sort of a duplication of what's happening in `request.js` on [line 14](https://github.com/Shopify/theme-scripts/pull/140/files#diff-377c1b6176cbb7f2e2a414384e5e81e8af6b9bf4efb1450b9e055187b10b174bR14):

``` 
var route = searchUrl ? searchUrl : '/search/suggest.json';
```

I'm wondering if we should be passing in the full url - including `/suggest.js` or just the [`routes.search_url`](https://shopify.dev/docs/themes/liquid/reference/objects/routes#routes-search_url) provided by shopify to the theme. As a theme dev, it's very easy to access that, and we could add `suggest.js` here or in request. What do you think? 

Maybe something like: 
Request.js
```
var route = searchUrl + '/suggest.json';
```

theme-predictive-search.js 
```  
  this.searchUrl = configParameters.search_url || '/search';
  if (configParameters.search_url) delete configParameters['search_url'];
 
  ...
}

PredictiveSearch.SEARCH_URL = "/search";
```

Reply:
Great suggestion! What I did is renaming `search_url` to `search_path` as we're now partially adding a "path" to the predictive search route instead of the full url `/search/suggest.json`